versions.tf

```diff
@@ -7,39 +7,35 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "4.38.0"
+      version = "4.63.1"
     }
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = "4.38.0"
+      version = "4.63.1"
     }
     kubernetes = {
       source  = "hashicorp/kubernetes"
-      version = "2.14.0" # Constrained by Google
+      version = "2.20.0" # Constrained by Google
     }
     local = {
       source  = "hashicorp/local"
-      version = "2.2.3"
-    }
-    template = {
-      source  = "hashicorp/template"
-      version = "2.2.0"
+      version = "2.4.0"
     }
     random = {
       source  = "hashicorp/random"
-      version = "3.1.0" # Constrained by Google
+      version = "3.5.1" # Constrained by Google
     }
     null = {
       source  = "hashicorp/null"
-      version = "3.1.0" # Constrained by Google
+      version = "3.2.1" # Constrained by Google
     }
     external = {
       source  = "hashicorp/external"
       version = "2.2.2" # Constrained by Google
```

Comment:
Should hashicorp/external version be updated to 2.3.1 per your table?

Reply:
It looks like hashicorp/null is 3.2.1 in my table, you may have gotten it mixed up with hashicorp/external which is below that.

Reply:
It's just the value of "2.2.2" on line 34 that I was asking about, probably should have removed the previous lines to avoid any confusion, sorry.

Here's the reference to hashicorp/external "2.3.1" from your table that I was asking about.

hashicorp/external
Versions:
Initial Version: 2.2.2
Final Version: 2.3.1
Notes:
No notable breaking changes or deprecations observed that would affect us

Reply:
Oh I see what you are pointing out now.
Let me fix that and get it through another quick test round. hashicorp/external is just used to generate the build info configmap so I can test this out quickly just by creating infrastructure without a viya deployment.